### ISSUE TITLE
NewFromString loss precsion with zero trailing

### ISSUE THREAD

**Issue by guotie** [shopspring/decimal#181]
if I use `NewFromString` to convert a string like  100.000, the decimal i got precision is 0!

for example:

```
ss := "100.000"
d, _ := decimal.NewFromString(ss)
fmt.Println(-d.Exponent())  // here is 0, but should be 3!
```

**Comment by mwoss** [comment]
Hi @guotie. Are you sure that you are using the latest decimal library version? The issue with truncated zeroes was fixed in PR #159.
I tried to run your code sample on version 1.2.0 and it works as supposed to.

**Comment by guotie** [comment]
yes, my problem. my version is v0.0.0-20200105231215-408a2507e114
